src/django/philly/templates/philly/map.html

```diff
@@ -0,0 +1,38 @@
+{% extends "philly/base.html" %}
+
+{% block content %}
+
+<h1>Philadelphia neighborhoods </h1>
+ <div id="mapid"></div>
+<style>
+    #mapid {
+        height: 600px;
+        margin: auto;
+        width: 800px;
+    }
+    h1 {
+
+    }
+</style>
+<script type="text/javascript">
+    var gj = '{{ data|escapejs }}';
+    var mymap = L.map('mapid').setView([39.95, -75.16], 11);
+    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+        maxZoom: 19,
+        attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+    }).addTo(mymap);
+
+    console.log(gj)
+    var geojson = jQuery.parseJSON(gj);
```

Comment:
Is jQuery necessary here? Could you use the built-in [JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse) and remove the jQuery dependency?